packages/esm-patient-search-app/src/patient-search-page/patient-banner/banner/patient-banner.scss

```diff
@@ -9,6 +10,36 @@
   grid-template-columns: 1fr auto;
 }
 
+.activePatientContainer {
+  background-color: $ui-02;
+}
+
+.deceasedPatientContainer {
+  background-color: colors.$gray-80;
```

Comment:
Maybe: `$color-gray-70` instead?

Reply:
Just realized that with my latest commit, I've basically undone this suggestion. The [design](https://zeroheight.com/23a080e38/p/6663f3-patient-header) specifies #393939 which [maps to](https://carbon-elements.netlify.app/colors/examples/preview/) `$colors.gray-80`.